containers/account/ChangePasswordModal.js

```diff
@@ -0,0 +1,394 @@
+            };
+        }
+
+        if (mode === MODES.CHANGE_TWO_PASSWORD_MAILBOX_MODE) {
+            return {
+                title: c('Title').t`Change mailbox password`,
+                labels: {
+                    oldPassword: c('Label').t`Old login password`,
+                    newPassword: c('Label').t`New mailbox password`,
+                    confirmPassword: c('Label').t`Confirm mailbox password`
+                },
+                onSubmit
+            };
+        }
+    })();
+
+    const isLoadingKeys =
+        loadingAddresses ||
+        loadingUserSettings ||
+        loadingOrganization ||
+        loadingOrganizationKey ||
+        loadingUserKeys ||
+        loadingAddressesKeys;
+
+    const eye = <Icon key="0" name="read" />;
+    const alert = (
+        <>
+            <Alert type="warning">
+                {c('Info')
+                    .t`Do NOT forget this password. If you forget it, you will not be able to login or decrypt your messages.`}
+                <br />
+                <br />
+                {c('Info')
+                    .jt`Save your password somewhere safe. Click on the ${eye} icon to confirm you that have typed your password correctly.`}
+                <br />
+                <br />
+                {c('Info')
+                    .t`We recommend adding a recovery email address first. Otherwise, you cannot recover your account if something goes wrong.`}
+            </Alert>
+        </>
+    );
+
+    const hasTotp = !!Enabled;
+
+    const children = isLoadingKeys ? (
+        <Loader />
+    ) : (
+        <>
+            {extraAlert}
+            {alert}
+            {!isSecondPhase && (
+                <Row>
+                    <Label>{labels.oldPassword}</Label>
+                    <Field>
+                        <PasswordInput
+                            value={inputs.oldPassword}
+                            onChange={({ target: { value } }) => setInput({ oldPassword: value })}
+                            error={errors.loginError}
+                            placeholder={c('Placeholder').t`Password`}
+                            required
+                        />
+                    </Field>
+                </Row>
+            )}
+            {!isSecondPhase && hasTotp && (
+                <Row>
+                    <Label>{c('Label').t`Two factor code`}</Label>
+                    <Field>
+                        <TwoFactorInput
+                            value={inputs.totp}
+                            onChange={({ target: { value } }) => setInput({ totp: value })}
+                            error={errors.loginError}
+                            placeholder={c('Placeholder').t`Two factor code`}
+                            required
+                        />
+                    </Field>
+                </Row>
+            )}
+            <Row>
+                <Label>{labels.newPassword}</Label>
+                <Field>
+                    <PasswordInput
```

Comment:
Since the user is supposed to enter a new password, `autocomplete` attribute should be set to `off`. Maybe we need to do something also on the `<form>`.